CONTRIBUTING.md

```diff
@@ -54,8 +53,10 @@ Nuget package will be created in root directory under \artifacts\packages\Debug
 
 ### Using the command line:
 
-Run e.g. `msbuild eng\mgmt.proj /t:"Runtests" /p:Scope=Compute`
-In the above example _RunTests_ will build and run tests for Compute only or you can use command line CLI
+Run e.g.\
```

Comment:
Why change this?

Reply:
It looked like it was intended to be on separate lines. Other `e.g.` statements in the document have the code below the statement.